server/jetty/src/main/java/io/deephaven/server/jetty/JettyBackedGrpcServer.java

```diff
@@ -122,13 +124,12 @@ public <T> T getEndpointInstance(Class<T> endpointClass) throws InstantiationExc
             });
         }
 
+        // Note: handler order matters due to pathSpec order
         HandlerCollection handlers = new HandlerCollection();
-
         // Set up /js-plugins/*
         JsPlugins.maybeAdd(handlers::addHandler);
-
+        // Set up /*
```

Comment:
/*?

Reply:
This is the handler for pathspecs matching `/*` - so everything that doesn't match `/js-plugins/*`.